No works with Angular 14 lastest

I cant use your library after update to `lastest@angular14`

```.sh
Your global Angular CLI version (15.0.4) is greater than your local version (14.2.10). The local Angular CLI version is used.

To disable this warning use "ng config -g cli.warnings.versionMismatch false".

     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/
    

Angular CLI: 14.2.10
Node: 16.17.0
Package Manager: npm 8.15.0 
OS: darwin arm64

Angular: 14.2.12
... animations, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router, service-worker

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1402.10
@angular-devkit/build-angular   14.2.10
@angular-devkit/core            14.2.10
@angular-devkit/schematics      14.2.10
@angular/cli                    14.2.10
@schematics/angular             14.2.10
rxjs                            6.6.7
typescript                      4.8.4
```

I am getting the following error when try to run `ng serve`

```.sh

Error: node_modules/ngx-scrolltop/lib/ngx-scrolltop.component.d.ts:21:18 - error TS2707: Generic type 'ɵɵComponentDeclaration' requires between 7 and 8 type arguments.

21     static ɵcmp: i0.ɵɵComponentDeclaration<NgxScrollTopComponent, "ngx-scrolltop", never, { "backgroundColor": "backgroundColor"; "symbolColor": "symbolColor"; "size": "size"; "symbol": "symbol"; "position": "position"; "theme": "theme"; "mode": "mode"; }, {}, never, ["*"], false, never>;
                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Error: node_modules/ngx-scrolltop/lib/ngx-scrolltop.directive.d.ts:17:18 - error TS2707: Generic type 'ɵɵDirectiveDeclaration' requires between 6 and 8 type arguments.

17     static ɵdir: i0.ɵɵDirectiveDeclaration<NgxScrollTopDirective, "[ngxScrollTop]", never, { "mode": "ngxScrollTopMode"; }, {}, never, never, false, never>;
                   
```

I think the version of the library is correct but no works

```.sh
% npm list ngx-scrolltop
my-app@2.2.1
└── ngx-scrolltop@5.0.2
```


my package.json

```.sh
{
  "name": "myapp",
  "version": "2.2.1",
  "scripts": {
    "ng": "ng",
    "start": "ng s -o",
    "build": "ng build",
    "test": "ng test",
    "lint": "ng lint",
    "e2e": "ng e2e",
    "postinstall": "ng build",
    "bundle-analizer": "ng build --stats-json && npx webpack-bundle-analyzer dist/myapp/stats.json",
    "bundle-dist-app": "ng build && node bundle-dist-config.js && start .",
    "sonar": "sonar-scanner",
    "test-code": "concurrently --raw \"ng test --watch --code-coverage\" \"node open-file-explorer\" ",
    "test-code-coverage": "ng test --watch --code-coverage",
    "bundle-dist-app-ci": "ng build && node bundle-dist-config.js",
    "test-code-ci": "npm run test -- --browsers ChromeHeadless --watch false --code-coverage false",
    "compodoc:build": "compodoc -p tsconfig.doc.json",
    "compodoc:build-and-serve": "compodoc -p tsconfig.doc.json -s",
    "compodoc:serve": "compodoc -s"
  },
  "private": true,
  "dependencies": {
    "@angular/animations": "^14.2.12",
    "@angular/common": "^14.2.12",
    "@angular/compiler": "^14.2.12",
    "@angular/core": "^14.2.12",
    "@angular/forms": "^14.2.12",
    "@angular/platform-browser": "^14.2.12",
    "@angular/platform-browser-dynamic": "^14.2.12",
    "@angular/router": "^14.2.12",
    "@angular/service-worker": "^14.2.12",
    "animate.css": "^4.1.1",
    "bootstrap": "^5.2.3",
    "bootswatch": "^5.1.3",
    "colors": "^1.4.0",
    "concurrently": "^7.2.2",
    "core-js": "^2.6.11",
    "date-fns": "^2.27.0",
    "express": "^4.17.1",
    "karma-coverage": "^2.2.0",
    "karma-mocha-reporter": "^2.2.5",
    "ng-lazyload-image": "^9.1.0",
    "ng-navigator-share": "^1.0.4",
    "ngx-masonry": "^13.0.0",
    "ngx-order-pipe": "^2.2.0",
    "ngx-pagination": "^5.0.0",
    "ngx-scrolltop": "^5.0.2",
    "ngx-stars": "^1.5.1",
    "ngx-toastr": "^12.1.0",
    "open": "^8.4.0",
    "rxjs": "^6.6.7",
    "tslib": "^1.11.1",
    "webpack-bundle-analyzer": "^4.5.0",
    "zip-lib": "^0.7.2",
    "zone.js": "^0.11.4"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "^14.2.10",
    "@angular/cli": "^14.2.10",
    "@angular/compiler-cli": "^14.2.12",
    "@angular/language-service": "^14.2.12",
    "@types/jasmine": "~2.8.8",
    "@types/jasminewd2": "^2.0.8",
    "@types/node": "^12.12.34",
    "jasmine-core": "^4.1.0",
    "jasmine-spec-reporter": "~4.2.1",
    "karma": "^6.3.19",
    "karma-chrome-launcher": "^3.1.1",
    "karma-coverage-istanbul-reporter": "^2.1.1",
    "karma-jasmine": "^4.0.2",
    "karma-jasmine-html-reporter": "^1.7.0",
    "protractor": "^7.0.0",
    "sonar-scanner": "^3.1.0",
    "ts-node": "~7.0.0",
    "tslint": "~6.1.3",
    "typescript": "~4.8.4"
  }
}

```

--- a/README.md
+++ b/README.md
@@ -44,15 +44,8 @@ Or play with it live on [stackblitz.com/edit/ngx-scrolltop](https://stackblitz.c
 > [See our Angular compatibility instructions…](#compatibility)
 
 ```bash
-ng add ngx-scrolltop # for Angular 12+ only! See our compatibility table
+ng add ngx-scrolltop # for latest Angular only! See our compatibility table
 ```
-
-<!--
-```bash
-npm install ngx-translate-cut # For Angular 13+. See compatibility table.
-```
-
-Choose the version corresponding to your Angular version: -->
 
 **Everything's done!** _(module imported and all settings automatically set in your project.)_
 
@@ -63,7 +56,7 @@ _Now just see some [options and examples](#options)._
 Via **yarn** or **npm**
 
 ```bash
-npm install ngx-scrolltop --save # for Angular 12+ only! See our compatibility table
+npm install ngx-scrolltop --save # for lastest Angular only! See our compatibility table
 
 # npm install ngx-scrolltop@2 --save # for Angular 5 – 11
 ```
@@ -95,7 +88,7 @@ If you still need to use it with an older version of Angular, then use version 2
 | Angular version | ngx-scrolltop | Install                       |
 | --------------- | ------------- | ----------------------------- |
 | ng15            | v5.x.x        | `ng add ngx-scrolltop@latest` |
-| ng14            | v5.x.x        | `ng add ngx-scrolltop@latest` |
+| ng14            | v4.x.x        | `ng add ngx-scrolltop@4`      |
 | ng13            | v4.x.x        | `ng add ngx-scrolltop@4`      |
 | ng12            | v4.x.x        | `ng add ngx-scrolltop@4`      |
 | ng5 – ng11      | v2.x.x        | `ng add ngx-scrolltop@2`      |
@@ -242,13 +235,35 @@ Easy! Use compatibility version of this library `npm install ngx-scrolltop@2 --s
 
 [See our Angular compatibility instructions…](#compatibility)
 
+#### Error
+
+If you are using Angular 14 with latest ngx-scrolltop library, you will see this error on build or start:
+
+```bash
+Error: node_modules/ngx-scrolltop/lib/ngx-scrolltop.component.d.ts:21:18 - error TS2707: Generic type 'ɵɵComponentDeclaration' requires between 7 and 8 type arguments.
+
+21     static ɵcmp: i0.ɵɵComponentDeclaration<NgxScrollTopComponent, "ngx-scrolltop", never, { "backgroundColor": "backgroundColor"; "symbolColor": "symbolColor"; "size": "size"; "symbol": "symbol"; "position": "position"; "theme": "theme"; "mode": "mode"; }, {}, never, ["*"], false, never>;
+                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+Error: node_modules/ngx-scrolltop/lib/ngx-scrolltop.directive.d.ts:17:18 - error TS2707: Generic type 'ɵɵDirectiveDeclaration' requires between 6 and 8 type arguments.
+
+17     static ɵdir: i0.ɵɵDirectiveDeclaration<NgxScrollTopDirective, "[ngxScrollTop]", never, { "mode": "ngxScrollTopMode"; }, {}, never, never, false, never>;
+```
+
+#### Solution
+
+Easy! Use compatibility version of this library `npm install ngx-scrolltop@4 --save`
+
+[See our Angular compatibility instructions…](#compatibility)
+
 ## Donation
 
 If this project have helped you save time please consider [making a donation](https://github.com/sponsors/bartholomej) for some 🍺 or 🍵 ;)
 
 ## License
 
-Copyright &copy; 2022 [Lukas Bartak](http://bartweb.cz)
+Copyright &copy; 2023 [Lukas Bartak](http://bartweb.cz)
 
 Proudly powered by nature 🗻, wind 💨, tea 🍵 and beer 🍺 ;)
 

